docs/sources/administration/data-source-management/index.md

```diff
@@ -127,7 +127,7 @@ You can make a panel retrieve fresh data more frequently by increasing the **Max
 
 ### Data sources that work with query caching
 
-Query caching works for all [Enterprise data sources](https://grafana.com/grafana/plugins/?type=datasource&enterprise=1) as well as the following [built-in data sources]({{< relref "../../datasources/" >}}):
+Query caching works for all [Enterprise data sources](/grafana/plugins/?type=datasource&enterprise=1) as well as the following [built-in data sources]({{< relref "../../datasources/" >}}):
```

Comment:
This link is failing in the preview, but that's supposed to be the case, right?  I'm only able to test this after it's merged and a complete website build is complete, right?